addons/website_slides/models/slide_channel.py

```diff
@@ -401,8 +401,10 @@ def toggle_active(self):
         to_archive = self.filtered(lambda channel: channel.active)
         to_activate = self.filtered(lambda channel: not channel.active)
         res = super(Channel, self).toggle_active()
+        super(Channel, self).write({'is_published': False})
```

Comment:
Why setting a channel as not published if you activate it for example ? Shouldn't you unpublish only de-activated channels ?

By the way why are you calling super().write and not directly write ?

Reply:
> Why setting a channel as not published if you activate it for example ? Shouldn't you unpublish only de-activated channels ?

I have updated the code to take care of this mistake. 

> By the way why are you calling super().write and not directly write ?

I just took the code that was already working for the toggle_active and adapted it.
How would have you done it ?

Reply:
Maybe it was to avoid loops ... try to see if it has any effect. Better ensure we don't have hidden loops :) .

Reply:
I removed the `super` function and used `self`.